### ISSUE TITLE
Go TLS CA - x509: certificate signed by unknown authority

### ISSUE THREAD

**Issue by gdhgdhgdh** [rebuy-de/aws-nuke#269]
Hi,

I recently used 2.5.0 to successfully nuke an account, and I'm trying the same thing on a different account today, but am bumping into this:

```
09:37 $ ./aws-nuke-v2.5.0-darwin-amd64 -c integr-prd.yml --access-key-id $AWS_ACCESS_KEY_ID --secret-access-key $AWS_SECRET_ACCESS_KEY --session-token $AWS_SESSION_TOKEN
Error: RequestError: send request failed
caused by: Post https://sts.amazonaws.com/: x509: certificate signed by unknown authority
```

As it turns out, the network setup here (large enterprise) is injecting its own CA into the middle of each TLS request:

```
09:40 $ curl -v https://sts.amazonaws.com/
*   Trying 54.239.21.217...
* TCP_NODELAY set
* Connected to sts.amazonaws.com (54.239.21.217) port 443 (#0)
* ALPN, offering h2

[ .. snip! ...]

* SSL connection using TLSv1.2 / AES256-GCM-SHA384
* ALPN, server did not agree to a protocol
* Server certificate:
*  subject: CN=sts.amazonaws.com
*  start date: Nov 10 00:00:00 2017 GMT
*  expire date: Nov 10 12:00:00 2018 GMT
*  subjectAltName: host "sts.amazonaws.com" matched cert's "sts.amazonaws.com"
*  issuer: C=US; ST=California; O=Zscaler Inc.; OU=Zscaler Inc.; CN=Zscaler Intermediate Root CA (zscloud.net); emailAddress=support@zscaler.com
*  SSL certificate verify ok.
```

There might have been some network change in the last week or so, but the interesting thing for me is that `curl` is working fine with whatever changes have been made to the root CA list on this managed Mac, but a Go-compiled binary isn't.

Do you know how I can inject / point Go towards the correct CA root cert?

Cheers,
Gavin.

**Comment by svenwltr** [comment]
Hello,

this fails, because some proxy breaks up your SSL connection (something from `Zscaler Inc.`). Either this is a company proxy or a MitM attack.

Golang usually uses the certificated provided by the operating system, but I think it is possible that the [`aws-sdk-go`](https://github.com/aws/aws-sdk-go) actually pins the certificate.

**Comment by gdhgdhgdh** [comment]
Yeah, ZScaler is the app we use for VPN (and content inspection and other tasteful things). I'll use another machine which isn't on the VPN to solve the immediate problem. 

Do you know of any way to pass a (and my heart sinks when I say this... ) 'ignore certs' flag?

**Comment by svenwltr** [comment]
I did not find anything about this in the Go SDK documentation.
